lang/semgrep-grammars/src/semgrep-move-on-sui/grammar.js

```diff
@@ -0,0 +1,29 @@
+/*
+  semgrep-move-on-sui
+
+  Extends the standard move grammar with semgrep pattern constructs.
+*/
+
+const base_grammar = require('tree-sitter-move-on-sui/grammar');
+
+module.exports = grammar(base_grammar, {
+  name: 'move',
```

Comment:
this needed to be move_on_sui, otherwise
./test-lang move-on-sui was failing with an OCaml linking error.